fabric-biome-api-v1/src/main/java/net/fabricmc/fabric/impl/biome/TheEndBiomeData.java

```diff
@@ -90,6 +90,30 @@ public static void addEndBarrensReplacement(RegistryKey<Biome> highlands, Regist
 		ADDED_BIOMES.add(barrens);
 	}
 
+	/**
+	 * Returns true if the given biome was added in the end, considering the Vanilla end biomes,
+	 * and any biomes added to the End by mods.
+	 */
+	public static boolean isIntendedForEndBiome(RegistryKey<Biome> biome){
+		return END_BIOMES_MAP.containsKey(biome);
+	}
+
+	/**
+	 * Returns true if the given biome was added as midland biome in the end, considering the Vanilla end biomes,
+	 * and any biomes added to the End as midland biome by mods.
+	 */
+	public static boolean isIntendedForEndMidlands(RegistryKey<Biome> biome){
+		return END_MIDLANDS_MAP.containsKey(biome);
+	}
+
+	/**
+	 * Returns true if the given biome was added as barrens biome in the end, considering the Vanilla end biomes,
+	 * and any biomes added to the End as barrens biome by mods.
+	 */
+	public static boolean isIntendedForEndBarrens(RegistryKey<Biome> biome){
+		return END_BARRENS_MAP.containsKey(biome);
```

Comment:
This doesn't seem to be the correct way to do it, the midlands and barrens map has the *highland* biome as the key.

Reply:
Yes, just realized that here quiqueck/BCLib#13